src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,10 +10,10 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_TELEGRAM = new Prefix("te/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-    public static final Prefix PREFIX_ZOOM_LINK = new Prefix("l/");
-    public static final Prefix PREFIX_ASSIGNMENT_NAME = new Prefix("a/");
-    public static final Prefix PREFIX_ASSIGNMENT_PERCENTAGE = new Prefix("%/");
-    public static final Prefix PREFIX_ASSIGNMENT_RESULT = new Prefix("r/");
+    public static final Prefix PREFIX_ZOOM_LINK = new Prefix("z/");
```

Comment:
Yup, I think it's a good idea to change the prefix to `z` because it represents more that it is the prefix for **z**oomlink 😄

Reply:
Yes, I thought the initial prefix "l/" would look confusing. Depending on the font used, it may look like the number 1 or the alphabet "I" as well.